### PR TITLE
Translate about logo and website (id)

### PR DESCRIPTION
**Pull request by meisyal**
I translated about logo and website in Bahasa Indonesia. Beside that, I updated **_config.xml** translation (site links, sidebar, footer, and credit). Please, review my pull request @gozali @stomar!

Thank you.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1073,11 +1073,11 @@ locales:
           url: /id/community/ruby-core/
           description: |
             Silakan berkontribusi untuk versi Ruby terbaru.
-#       issue_tracking:
-#         text: Issue Tracking
-#         url: https://bugs.ruby-lang.org/
-#         description: |
-#           Report or help solve issues in Ruby.
+        issue_tracking:
+          text: Laporkan Masalah
```
Comment (gozali) [inline _config.yml]: remove whitespace here, before `text`

Comment (meisyal) [inline _config.yml]: Where is that? I didn't see whitespace before `text`?

Comment (gozali) [inline _config.yml]: Sorry, false alarm 😁. I didn't see removed # character

**Comment by gozali** [comment]
All good. +1

File: _config.yml [hunk 2]
```diff
@@ -1849,11 +1850,8 @@ locales:
       ou contactez le <a href="mailto:webmaster@ruby-lang.org">webmaster</a>
       pour toute question ou tout commentaire.
     id: |
-      Situs ini dikembangkan menggunakan bahasa Ruby dan <a href="http://www.jekyllrb.com/">Jekyll</a>.
-      Dipersembahkan dengan bangga oleh anggota komunitas Ruby.
-      Silakan berkontribusi melalui <a href="https://github.com/ruby/www.ruby-lang.org/">GitHub</a>
-      atau hubungi <a href="mailto:webmaster@ruby-lang.org">webmaster</a> kami
-      untuk pertanyaan atau komentar mengenai situs ini.
+      <a href="id/about/website"/">Situs ini</a>
```
Comment (stomar) [inline _config.yml]: Wrong link (**/**id/...) has been fixed with d4c9c06c4a77e8d81826cd541a934cd670eec71b.

**Comment by stomar** [comment]
Thanks!

**Comment by meisyal** [comment]
Sorry, I didn't notice that. By the way, thank you for fixing and merging @stomar.
Thank you @gozali for reviewing this pull request.

**Pull request merged**